### PR TITLE
Changes cult desks/tables into tables + changes cult altar path

### PR DESCRIPTION
**Pull request by Firecage**
Changes cult desks/tables into tables + changes cult altar path to be less misleading.

:cl: Firecage
rscadd: Changes cult tables into real tables. Cultists can make them with a table frame+runed metal.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/table_frames.dm [hunk 1]
```diff
@@ -113,3 +113,18 @@
 			new /obj/structure/table/wood/poker(src.loc)
 			qdel(src)
 		return
+	if(istype(I, /obj/item/stack/sheet/runed_metal))
+		var/obj/item/stack/sheet/runed_metal/RM = I
+		if(!iscultist(user))
+			user << "<span class='notice'>You are unsure what the next step may be.</span>"
+			return
+		if(RM.get_amount() < 1)
+			user << "<span class='warning'>You need one carpet sheet to do this!</span>"
```
Comment (Firecage) [inline code/game/objects/structures/table_frames.dm]: Shit. WIll fix it as soon as I fix my master repo.

**Comment by MrStonedOne** [comment]
Changes game content and item interactions, as well as adding new content (buildable cult tables), Please make a changelog.

You can include a changelog in the body of the pr now: https://tgstation13.org/phpBB/viewtopic.php?f=5&t=5157

**Comment by Anonus** [comment]
Technically they're not buildable as you can't get runed metal from anywhere and only cultists can do it anyway.

**Comment by Aranclanos** [comment]
antag checks for these kind of things are pretty bad, just let anyone be able to build them

**Comment by Firecage** [comment]
@Aranclanos But then for consistency sake, should I then also let anyone be able to use runed metal to build the other useless cult structures?

**Comment by Anonus** [comment]
Runed metal should be cultist only, I might actually do something with it.

**Comment by Aranclanos** [comment]
yeah sure, some love for sandbox servers is always nice

**Comment by Firecage** [comment]
@Aranclanos Well, will be impossible for me to do either right now....what with my branch somehow disappearing.

**Comment by Cheridan** [comment]
Uh just pull from your remote. or just use the github browser editor.

**Comment by Firecage** [comment]
Cheridan, the problem from pulling from the remote is....theres no remote to pull from.

**Comment by Cheridan** [comment]
oh

huh

what the heck did you doooo

**Comment by Firecage** [comment]
....Beats me. My branch went bye bye.

**Comment by Dorsisdwarf** [comment]
You call that an oh god mystery error? Hah!

**Comment by Aranclanos** [comment]
just make a new pr if you can't touch this one

**Comment by Firecage** [comment]
Eh, I'le remake the PR later.

**Comment by Aranclanos** [comment]
ok

**Pull request closed without merging**